src/app/hearing/components/view-hearing/view-hearing.component.scss

```
@@ -0,0 +1,51 @@
+mat-card {
+  max-width: 900px;
+  margin: 0 auto;
+}
+
+.left {
+  float: left;
+}
+
+.note .content {
```

tip: you could use sass like
```suggestion
.note {
    .content {
        width: 600px;
    }
    
    .author-info {
        width: 300px;
    }
}
```

---

thanks, I didn't know that